WordPress/WordPressUITests/Screens/Editor/EditorScreen.swift

```diff
@@ -17,24 +17,34 @@ class EditorScreen: BaseScreen {
     private var richTextField = "Rich Content"
     private var htmlTextField = "HTMLContentView"
 
-    let titleView = XCUIApplication().textViews["Title"]
+    let editorCloseButton = XCUIApplication().navigationBars["Azctec Editor Navigation Bar"].buttons["Close"]
+    let publishButton = XCUIApplication().buttons["Publish"]
+    let moreButton = XCUIApplication().buttons["More"]
+    let uploadProgressBar = XCUIApplication().progressIndicators["Progress"]
 
-    lazy var mediaButton = XCUIApplication().buttons["formatToolbarInsertMedia"]
-    lazy var headerButton = XCUIApplication().buttons["formatToolbarSelectParagraphStyle"]
-    lazy var boldButton = XCUIApplication().buttons["formatToolbarToggleBold"]
-    lazy var italicButton = XCUIApplication().buttons["formatToolbarToggleItalic"]
-    lazy var underlineButton = XCUIApplication().buttons["formatToolbarToggleUnderline"]
-    lazy var strikethroughButton = XCUIApplication().buttons["formatToolbarToggleStrikethrough"]
-    lazy var blockquoteButton = XCUIApplication().buttons["formatToolbarToggleBlockquote"]
-    lazy var listButton = XCUIApplication().buttons["formatToolbarToggleListUnordered"]
-    lazy var linkButton = XCUIApplication().buttons["formatToolbarInsertLink"]
-    lazy var horizontalrulerButton = XCUIApplication().buttons["formatToolbarInsertHorizontalRuler"]
-    lazy var sourcecodeButton = XCUIApplication().buttons["formatToolbarToggleHtmlView"]
-    lazy var moreButton = XCUIApplication().buttons["formatToolbarInsertMore"]
+    let titleView = XCUIApplication().textViews["Title"]
+    let contentPlaceholder = XCUIApplication().staticTexts["aztec-content-placeholder"]
+
+    lazy var mediaButton = XCUIApplication().buttons["format_toolbar_insert_media"]
```

Comment:
How come these are marked `lazy var`? It seems like they could just be `let` instead – it's more consistent and a bit more readable :)

Reply:
All the toolbar buttons were declared with `lazy var` in the previous UI tests that I inherited. My assumption was that it was done to be able to have them all defined so they're ready to use, but wait to compute them until they are used (since we don't use most of them in our existing tests yet). That said, I have no strong opinion/experience with that and can certainly change them to `let` instead. (These will also change when we roll out the block editor.)